core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -280,5 +283,23 @@ public static boolean isUfsObjectStorage(String ufsAddress) {
         || ufsAddress.startsWith(Constants.HEADER_OSS);
   }
 
+  /**
+   * Gets the value from the static key=value mapping in string format. E.g. mapping
+   * "id1=user1;id2=user2" returns value "user1" with key "id1", returns value null with key
+   * "invalid". Returns null if the given key does not exist in the mapping.
+   *
+   * @param mapping the "key=value" mapping in string format separated by ";"
```

Comment:
The format for this util seems fairly specific, should it really be in CommonUtils?